apps/graduate/src/features/schedule/components/ScheduleEditModal.tsx

```diff
@@ -45,84 +47,84 @@ export default function ScheduleEditModal({
             }
           : item,
       ),
-    )
-    setIsModalOpen(false)
-  }
+    );
+    setIsModalOpen(false);
+  };
 
   const handleCancel = () => {
-    setIsModalOpen(false)
-  }
+    setIsModalOpen(false);
+  };
 
   const handleStageChange = (stage: string) => {
-    setSelectedStage(stage)
-    const found = scheduleData.find((item) => item.stage === stage)
-    setStartDate(found ? dayjs(found.startDate) : dayjs())
-    setEndDate(found ? dayjs(found.endDate) : dayjs())
-  }
+    setSelectedStage(stage);
+    const found = scheduleData.find(item => item.stage === stage);
+    setStartDate(found ? dayjs(found.startDate) : dayjs());
+    setEndDate(found ? dayjs(found.endDate) : dayjs());
+  };
 
   return (
     <>
       <div className={style.editButtonWrapper}>
-        <Button type="default" onClick={handleEdit} size="large">
-          수정
+        <Button type='default' onClick={handleEdit} size='large'>
+          ????
         </Button>
       </div>
       <Modal
```

Comment:
```suggestion
      <Modal
      centered
```

centered 속성을 주면 모달이 중앙 정렬 됩니다. 다른 모달들도 넣어주시면 좋겠어요